### ISSUE TITLE
panic:bad-yaml using README example

### ISSUE THREAD

**Issue by scnewma** [teaxyz/cli#384]
Trying out the README example for developer environments results in a panic.

Version:

```
❯ tea --version
tea 0.24.3
```

Reproduce:

```
$ mkdir my-project
$ cat <<EOF >>my-project/README.md
# Dependencies
| Project    | Version |
| ---------- | ------- |
| nodejs.org | ^18.4   |
EOF
$ cd my-project
$ tea -SE

Error: bad-yaml
    at usePackageYAML (file:///opt/tea.xyz/src-v0.24.3/src/hooks/usePackageYAML.ts:9:37)
    at refineFrontMatter (file:///opt/tea.xyz/src-v0.24.3/src/hooks/usePackageYAML.ts:30:16)
    at supp (file:///opt/tea.xyz/src-v0.24.3/src/hooks/useVirtualEnv.ts:109:20)
    at async default (file:///opt/tea.xyz/src-v0.24.3/src/hooks/useVirtualEnv.ts:24:13)
    at async file:///opt/tea.xyz/src-v0.24.3/src/app.ts:24:21
```

**Comment by vladdeSV** [comment]
Cannot reproduce on Linux (Arch)

```
$ tea --version
tea 0.24.3

$ mkdir my-project

$ cat <<EOF >>my-project/README.md
# Dependencies
| Project    | Version |
| ---------- | ------- |
| nodejs.org | ^18.4   |
EOF

$ cd my-project

$ tea -SE
pantries sync’d ⎷
# snip...
```

**Comment by scnewma** [comment]
After more investigation, `tea` is actually panicing because of what was in the parent directory to my-project. Closing this issue.

**Comment by mxcl** [comment]
> tea is actually panicing because of what was in the parent directory to my-project. Closing this issue.

Sounds like a bug still. What was there?

**Comment by scnewma** [comment]
You can reproduce in a fresh deno project.

```
$ tea +deno.land deno init my-project
$ cd my-project
$ tea -E

Error: bad-yaml - usePackageYAML
    at usePackageYAML (file:///Users/shaun/dev/tea/src/hooks/usePackageYAML.ts:9:37)
    at refineFrontMatter (file:///Users/shaun/dev/tea/src/hooks/usePackageYAML.ts:30:16)
    at supp (file:///Users/shaun/dev/tea/src/hooks/useVirtualEnv.ts:109:20)
    at async default (file:///Users/shaun/dev/tea/src/hooks/useVirtualEnv.ts:24:13)
    at async file:///Users/shaun/dev/tea/src/app.ts:24:21
```

Version info:

```
$ tea +deno.land deno --version
deno 1.30.3 (release, x86_64-apple-darwin)
v8 10.9.194.5
typescript 4.9.4
```

**Comment by mxcl** [comment]
This should be fixed with 0.24.5. LMK!

**Comment by scnewma** [comment]
Yep. Looks good! Thanks!
